quote-book/src/synchronized_quote_book.rs

```diff
@@ -35,7 +108,7 @@ where
     ) -> Result<Quote, QuoteBookError> {
         // Check to see if the current_block_index is already at or past the
         // max_tombstone_block for the sci.
-        let current_block_index = self.ledger.num_blocks()? - 1;
+        let current_block_index = self.get_current_block_index();
```

Comment:
Whats the reason for this? No point in adding a quote we know is going to get removed once the ledger scan has reached it.

Reply:
Yes. You're right. I don't know what I was thinking.